### ISSUE TITLE
Add operators to the charging_state attribute

### ISSUE THREAD

**Issue by darylvL** [maxwroc/battery-state-card#254]
**Is your feature request related to a problem? Please describe.**
One of my devices only exposes whether or not it is charging trough it's icon. This icon can be, depending on the current battery level, any of the following: mdi:battery-charging-100, mdi:battery-charging-90, mdi:battery-charging-80,...
I'd like to use the many possibilities your card provides while also being able to see whether or not my device is charging.

**Describe the solution you'd like**
I'd like to be able to use operators (like the ones in filter objects) on the attribute object of the charging state object.
This would allow me to check whether the icon contains the word "charging" and show the correct icon.

**Additional context**
The config would look like this:
```
...
entities:
  - entity: sensor.my_device_battery_level
    charging_state:
      attribute:
        name: icon
        operator: contains
        value: 'charging'
...
```

**Comment by maxwroc** [comment]
Closing for now as this #347 should solve this issue I think. Please let me know if it's not sufficient. This will be available in v2.0.0

**Comment by darylvL** [comment]
That solves my issue indeed, thanks!
